api/src/domain/updateSidebarCards.test.ts

```diff
@@ -34,14 +34,35 @@ describe("updateRoadmapSidebarCards", () => {
       );
     });
 
+    it("removes not-registered-existing-account card and adds formation nudge card", async () => {
+      const userData = generateUserDataForBusiness(
+        generateBusiness({
+          profileData: generateProfileData({
+            operatingPhase: "NEEDS_TO_FORM",
+          }),
+          preferences: generatePreferences({
+            visibleSidebarCards: ["not-registered-existing-account"],
+          }),
+        })
+      );
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(getCurrentBusiness(updatedUserData).preferences.visibleSidebarCards).not.toContain(
+        "not-registered-existing-account"
+      );
+      expect(getCurrentBusiness(updatedUserData).preferences.visibleSidebarCards).toContain(
+        "formation-nudge"
+      );
+    });
+
```

Comment:
[dust] Do we need this test given on line 80 we check for this exact logic, and that no other cards are modified? I understand wanting to be explicit about the behavior, but these seems like one test could cover everything here.

Reply:
It's the same test but for the new different card. How would you suggest covering it all in one test?

Reply:
The only difference is that the second test has an additional card `other-card`. I'm saying the coverage is already there. In both cases we're validating that `not-registered-existing-account` is replaced with `formation-nudge`. Unless I'm missing something we already are covering it all in one test.

Reply:
Oh wait I looked at the wrong line. That's true but the reason for separating was to describe the 2 different pieces of behavior. I think the test name would be really long and wordy if it was all one test

Reply:
Totally fair.